srcpkgs/tzdata/template

```diff
@@ -2,7 +2,7 @@
 pkgname=tzdata
 version=2021a
 revision=1
-wrksrc=tzdata
+bootstrap=yes
```

Comment:
Removing `wrksrc` seems wrong to me...

Reply:
It's `create_wrksrc=yes`, so remove `wrksrc=` is fine.